### PR TITLE
Fixes issue with docker

### LINKED ISSUE #350: docker-compose up fails with a bundler error
It seems that Gemfile and Dockerfile don't agree in their bundler version which is causing problems.

Trace:

```
 > [5/6] RUN bundle install:
#10 0.317 Warning: the running version of Bundler (1.16.0) is older than the version that created the lockfile (1.16.6). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
#10 0.322 Your Ruby version is 2.4.2, but your Gemfile specified 2.4.4
------
executor failed running [/bin/sh -c bundle install]: exit code: 18
ERROR: Service 'web' failed to build : Build failed
```

I can submit a patch for this.

### PR DESCRIPTION
**Pull request by etagwerker**
Hi there,

This PR is an attempt to fix #350 so that it works once again. 

Please check it out.

Thanks!

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,10 +6,11 @@ services:
     - 27017:27017
 
   web:
+    platform: linux/amd64
```
Comment (potomak) [inline docker-compose.yml:9]: Is this change really necessary?
What is the error if you don’t specify the platform?

File: docker-compose.yml [hunk 2]
```diff
@@ -6,10 +6,11 @@ services:
...
     build: .
     volumes:
       - .:/tomatoes
-    entrypoint: ./docker-entrypoint.sh
```
Comment (potomak) [inline docker-compose.yml:12]: This change isn’t strictly required and it could be excluded from the PR

**Comment by potomak** [comment]
Thanks for making this change!

Is the platform field really required? Can you share more about the error that you get if you don’t specify it please?

**Review by lubc — APPROVED** [review]
